packages/client/src/__tests__/integration/happy/exhaustive-schema-mongo/__snapshots__/test.ts.snap

```diff
@@ -1485,8 +1490,14 @@ export namespace Prisma {
   export type Middleware<T = any> = (
     params: MiddlewareParams,
     next: (params: MiddlewareParams) => Promise<T>,
+    context: QueryMiddlewareContext
   ) => Promise<T>
 
+  /**
+   * Context type
+   */
+  export type QueryMiddlewareContext = Record<string, string | number | boolean>;
```

Comment:
Would like to have `string[]` as well, e.g. for user roles (could also do that as a `number` with bitwise operators, but why not).